src/dbridge_js/ditecx/dsig-xades-bp-adapter.ts

```diff
@@ -157,4 +157,13 @@ export class DSigXadesBpAdapter extends DSigAdapter {
       callback
     );
   }
+
+  setWindowSize(
```

Comment:
Toto volá JS na ORSR portáli. Dobre som to pridal?

Reply:
hej toto vyzera dobre